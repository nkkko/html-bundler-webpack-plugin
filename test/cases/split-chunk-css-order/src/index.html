<!DOCTYPE html>
<html>
<head>
  <script src="./main.js" defer></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>