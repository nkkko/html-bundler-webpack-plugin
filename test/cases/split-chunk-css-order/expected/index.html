<!DOCTYPE html>
<html>
<head>
  <link href="main.bundle.css" rel="stylesheet">
<script src="main.bundle.js" defer></script>
</head>
<body>
  <h1>Hello World!</h1>
</body>
</html>